### ISSUE TITLE
[Bug] Unable to make regular links in bot long description

### ISSUE THREAD

**Issue by PikaDude** [discordlist-gg/beta-issues#34]
## Description
I'm trying to make a simple hyperlinked heading, and I'm unable to do so as it just turns into a button.
![image](https://user-images.githubusercontent.com/18642415/131257503-6b130334-6d09-411d-9aac-41107cc02add.png)

## Possible Solution
Make the buttons optional or require a class.

## Steps to Reproduce (for bugs)
```html
<h1><a href="https://erisly.com">Click here to visit my full website!</a></h1>
````

**Comment by audn** [comment]
All links are converted into buttons at the moment. Do you think this is a bad idea?

**Comment by retraigo** [comment]
It could be a bad idea when you want inline links instead of buttons

**Comment by PikaDude** [comment]
> All links are converted into buttons at the moment. Do you think this is a bad idea?
yes as @retraigo says, people do like to use inline links, automatically assuming they don't and enforcing buttons is not a good idea

**Comment by audn** [comment]
I think it's a cool idea, so I'll see if I can figure out a way to choose between inline / button 😄
